README.md

```diff
@@ -5,77 +5,87 @@
 
 **Looking for your help!**
 
-For Rails Girls Summer of Code we are planning to build a simple app that
-aggregates daily status updates, commit activity, GitHub issues and other
-things into an activity stream.
+For RSGoC we are building an app that aggregates daily status updates, commit activity and other sources into an activity stream.
 
 Main goals are:
 
-* make it easy to get an overview of activity/progress for each of the teams for supervision
-* make it easy for interested remote coaches to find opportunities to give support/help
+* make it easy to get an overview of each team's activity/progress for supervision
+* make it easy for people interested in coaching to find opportunities to give support/help
 * display to the world how much amazing stuff is happening
 
-We are planning to require teams to keep a daily log of short updates about
-their work. Our idea is to allow any sort of blog type tool for that (maybe
-recommend a few) and aggregate things through RSS in a central app. This app
-could then act as a webhook target for GitHub events.
-
-Since we still are somewhat overwhelmed with the amount of work we'd like to
-ask the community for help with this. The app would need to be available
-(initial, basic version) on 1st of July, ideally a few days earlier.
-
 Features:
 
-* Users can sign in through GitHub Oauth
-* They can create and update teams
-* Teams have members (students, coaches, mentors aka project maintainers), GitHub repositories, a log URL
-
+* Users can sign in through GitHub OAuth
+* Students can create teams and apply for the program
+* Teams can organize their members (students, coaches, mentors aka project maintainers)
 * RSS feeds are fetched from all teams' logs regularly and aggregated
-* There is a webhook endpoint for GitHub events that aggregates information about issues, pull requests and such
+* Organizers can process new projects for submission, review new teams for selection, keep track of information on conferences for students
+
+We're really excited at how much this app has grown and developed, and are really
+grateful for all of the community help along the way. There's still plenty to do
+here - and we'd love to have your contributions! Hop on over to the
+[issues](https://github.com/rails-girls-summer-of-code/rgsoc-teams/issues)
+and have a look 👀.
 
 Requirements:
 
-* Keep it simple, so Rails Girls students can get involved, too
+* Keep it simple to lower the barrier for RGSoC students to contribute too
 * By contributing, you agree to adhere to our [Code of Conduct](CODE_OF_CONDUCT.md)
 * Make sure to check our [Contribution Guide](CONTRIBUTING.md)
 
-## System Requirements
+## Getting Started
+
+In order to contribute, you need to run the Teams App locally on your computer. For setting it up, you'll need to have some tools installed on your system:
 
 * PostgreSQL 9.5 or newer
 * Ruby 2.5.1
 * [Chromedriver](https://sites.google.com/a/chromium.org/chromedriver/)
 
```

Comment:
Rails may come in handy too ;-)
Shall we put the system requirements at the top of the file? I see that a lot, I like that. 
And keep the set up of those in the Installation sections.

Reply:
> Rails may come in handy too ;-)

☝️ why? The app needs a particular version _(that will be installed as a dependency when `bundle install`)_ - so I don't really see a point in people installing a potentially non-matching version first. Or did I get your point wrong?

For putting the requirements to the top of the file: this makes 💯 sense for libraries and other things that you'd run as a dependency. For the sake of readability in our case, I'd personally keep it close to the _"I want to setup / work with this app locally"_ to not scatter the information into different corners of the README.

Reply:
I think it's good PR to show potentially new contributors that we are on a 5+ rails version, and from my experience, it saves time if the project shows that upfront in the Readme.
I see that it is not a 'system requirement'; maybe we can add a online 'stack' heading? Up to you.

Reply:
Ok, got your point, thanx 🙏 
As every character we add to the README is actually code that needs to be maintained, I chose to however not add it 😉 The _stack_ section would need to be kept up to date and in sync with the rest of the README. Looking at the very first introductory section which was out of date for years shows that this is a hard thing to do 🙈